src/constants/systemIntake.ts

```diff
@@ -1,3 +1,4 @@
 export const GET_SYSTEM_INTAKE: string = 'GET_SYSTEM_INTAKE';
 export const PUT_SYSTEM_INTAKE: string = 'PUT_SYSTEM_INTAKE';
 export const DELETE_SYSTEM_INTAKE: string = 'DELETE_SYSTEM_INTAKE';
+export const UPDATE_SYSTEM_INTAKE: string = 'UPDATE_SYSTEM_INTAKE';
```

Comment:
I know didn't add `GET_SYSTEM_INTAKE` or `PUT_SYSTEM_INTAKE`, but should we standardize on naming this constants so that they're consistent? @Igarfinkle started using `FETCH_` to indicate fetching from API and `STORE_` putting what is received in the API into Redux.

In that same light, should we name this `STORE_SYSTEM_INTAKE`?

Reply:
Yes, I think we should use `FETCH` and `STORE` for redux actions, and `GET` and `PUT` for sagas actions.

Reply:
Well, hang on. Fetch is already a keyword in api requests. Let's use `RETRIEVE` and `STORE` for the Redux store. Is that too wordy @kahlouie @christopherhuii

Reply:
Why can't be use `FETCH_SYSTEM_INTAKE`?

Reply:
I ended up using `GET` `PUT` for saga actions, `RETRIEVE` and `STORE` for reducer actions and `FETCH` for the effect function name in the view.